package.json

```diff
@@ -60,7 +60,7 @@
     "pretty:ui": "cd ./packages/ui && yarn pretty",
     "preversion": "./packages/scripts/src/bash/release-pre-version.sh",
     "remote:build:data:zip": "node ./packages/scripts/src/js/packageJsonDataFiles.js",
-    "remote:build:index:specimen": "node ./packages/scripts/src/js/exec.js docker-rebuild-search.sh",
+    "remote:build:indices": "node ./packages/scripts/src/js/exec.js docker-rebuild-search.sh",
```

Comment:
maybe it should be called `remote:build:elastic-indices` for consistency (and to indicate it's not postgres indices)

Reply:
fixed